nats/aio/client.py

```diff
@@ -258,7 +259,13 @@ async def __anext__(self):
 class Msg:
```

Comment:
pre-existing: this would make good use case for a [dataclass](https://docs.python.org/3/library/dataclasses.html) potentially (available since python 3.6)